engines/tinsel/tinsel.cpp

```diff
@@ -1012,6 +1013,7 @@ Common::Error TinselEngine::run() {
 	_handle = new Handle();
 	_scroll = new Scroll();
 	_dialogs = new Dialogs();
+	_notebook = new Notebook();
```

Comment:
This is only necessary for Noir though, I guess we can assign it nullptr otherwise.